setup.py

```diff
@@ -12,6 +12,7 @@
         "frex.pipelines",
```

Comment:
Use setuptools find_packages or find_namespace_packages.

Reply:
updated to find_packages.